docker/Dockerfile_centos

```diff
@@ -23,7 +23,7 @@ COPY src ${AV_DEV}/src
 
 WORKDIR "${AV_BUILD}"
 
-RUN cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS:BOOL=ON -DTARGET_ARCHITECTURE=core \
+RUN cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS:BOOL=ON \
```

Comment:
We need to make TARGET_ARCHITECTURE an option of the docker image.
I need to use that to provide generic binaries for the releases that can work on any computer.

Reply:
Ok, I'll add an argument for additional cmake args.

I wonder if the dockerhub images should use something more recent than 'core' though? Eg. Haswell is 7 years old and enables a lot more vector capabilities, so there's more opportunity for the compiler to auto-vectorise.

Reply:
Yes, that's very good question.